systemtests/mesh_cli_config/testnet-construction.json

```diff
@@ -0,0 +1,35 @@
+{
+  "network": {
+    "blockchain": "MultiversX",
+    "network": "untitled"
+  },
+  "data_directory": "",
+  "http_timeout": 200,
+  "max_retries": 1,
+  "max_online_connections": 120,
+  "max_sync_concurrency": 1,
+  "construction": {
+    "end_conditions": {
+      "transfer": 1
+    },
+    "stale_depth": 10,
+    "broadcast_limit": 5,
+    "constructor_dsl_file": "testnet-construction.ros",
+    "prefunded_accounts": [
+      {
+        "account_identifier": {
+          "address": "erd1ldjsdetjvegjdnda0qw2h62kq6rpvrklkc5pw9zxm0nwulfhtyqqtyc4vq"
+        },
+        "privkey": "3e4e89e501eb542c12403fb15c52479e8721f2f4dedc3b3ef0f3b47b37de006c",
```

Comment:
The testing key is not hidden, but this shouldn't be an issue for now. In the future, we will have it as a GH secret.